site/pages/review-disruption.page.tsx

```diff
@@ -0,0 +1,409 @@
+                                            >
+                                                {`Consequence ${i + 1} - ${consequence["mode-of-transport"]} - ${
+                                                    consequence.service
+                                                        ? `Services - ${consequence["service"]
+                                                              .map((service) => service.id)
+                                                              .join(", ")}`
+                                                        : consequence["consequence-type"] ===
+                                                              ConsequenceType.operatorWide &&
+                                                          consequence["consequence-operator"]
+                                                        ? `${ConsequenceType.operatorWide} - ${consequence["consequence-operator"]}`
+                                                        : `${ConsequenceType.networkWide}`
+                                                }`}
+                                            </span>
+                                        </h2>
+                                    </div>
+                                    <div
+                                        id={`accordion-default-content-${i + 1}`}
+                                        className="govuk-accordion__section-content"
+                                        aria-labelledby={`accordion-default-heading-${i + 1}`}
+                                    >
+                                        <Table
+                                            rows={[
+                                                {
+                                                    header: "Mode of transport",
+                                                    cells: [
+                                                        consequence["mode-of-transport"],
+                                                        createChangeLink(
+                                                            "mode-of-transport",
+                                                            ADD_CONSEQUENCE_PAGE_PATH,
+                                                        ),
+                                                    ],
+                                                },
+                                                {
+                                                    header: "Consequence type",
+                                                    cells: [
+                                                        consequence["consequence-type"],
+                                                        createChangeLink("consequence-type", ADD_CONSEQUENCE_PAGE_PATH),
+                                                    ],
+                                                },
+                                                {
+                                                    header: "Service",
```

Comment:
Service(s) maybe

Reply:
Although I suppose the designs maybe take precedent - certainly we need (s) on the end

Reply:
done